README.md

```diff
@@ -1,91 +1,65 @@
 
-Note: This step only needs to be run the first time to create the stack. If you need to make updates please follow the
-instructions in [Update the content-distribution after changes](#update-the-content-distribution-after-changes).
+Note: The difference between a Lambda Function and a Lambda@Edge Function is that a Lambda@Edge function is associated with a CloudFront Distribution.
 
-Note: This command will take about 15-25min to complete. It will create the following resources in AWS:
+## Getting Started
+This stack is created and deployed with AWS CloudFormation. AWS CloudFormation gives us a single source of truth with a simple text file, in our case `cloudfront-single-origin-bucket.yaml`, to help us describe and provision our AWS cloud infrastructure as oppose to using the AWS Management Console directly.
 
-- Cloudfront distribution
-- Artifact S3 Bucket
-- Raw JSON S3 Bucket
-- Baked HTML S3 Bucket
-- Resources S3 Bucket
-- It will not create the [Lambda Function](./sam-app/lambda_function.py) because this requires the Artifact S3 to be created first.
+Reference: [AWS CloudFormation Documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/Welcome.html)
 
-You can see the progress of deployment in the AWS console [here](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks). Click on the stack name `content-distribution` and open the `events` tab.
+### Prerequisites
+You will need to have [AWS CLI](https://aws.amazon.com/cli/) to run commands for AWS CloudFormation.
 
-### Update the content-distribution after changes
+### Create Content Distribution Stack in AWS
+This step only needs to be run once to create the stack. 
+To make updates to the stack, see [Update Content Distribution After Changes](#update-content-distribution-after-changes).
 
-#### Build SAM app
-
-If you have done by now please build the SAM app:
+Create stack, run AWS CLI Command:
 
 ```bash
-cd sam-app
-# activate a Python3 virtualenv here!
-sam build -b ../.aws-sam
-cd ..
+aws cloudformation deploy \
+--stack-name cops-distribution \
+--template-file cf-templates/cloudfront-single-origin-bucket.yaml \
+--region us-east-1 \
+--tags Project=content-distribution Application=content-distribution Environment=dev Owner=ConEng \
+--capabilities CAPABILITY_IAM
 ```
+This command will take about 15-25min to complete. It will create the following resources in AWS:
 
-#### Package SAM app and upload
-
-The `aws cloudformation` command can package up the changes and upload to an s3 bucket.
-It allows you to do this by using the `package` argument along with `--s3-bucket` and `--output-template-file` options.
-
-This will package up the lambda function, upload to the s3 bucket, and generate a new template that has
-the s3 bucket substituted in the proper locations of the template.
+- AWS CloudFront Distribution
+- 2 AWS S3 Buckets:
+	- Artifacts Bucket: Resources for the CloudFront Distribution
```

Comment:
I may be mistaken, but I thought this artifacts bucket was used earlier for the lambda code packages. If so, is it still needed?

Reply:
Ah, you super are right, no it won't be needed anymore, will make the corrections.

Reply:
Note: Readme is updated but an update need to be made to the CloudFormation stack, safely.

Reply:
Note: Did not find a way a straight forward way to update the template and redeploy. 

Questions:
Unanswered: Does a redeploy need to happen if a template is modified?

Course of action:
Deleted the cops-distribution stack (which takes care of resources via Resource Deletion Policy)
Modified the template
Deploy (Like New) again to CloudFront distribution.

I didn't want to fight with it anymore.

Reply:
Just FYI to some of the points above:
- If you modify a template and then do an `update-stack` command it should take care of figuring out what's different and making the appropriate changes to the underlying resources
- For S3 buckets, `DeletionPolicy` determines whether CF will try to destroy the bucket if you bring down the stack, but no matter what if there's data it won't delete it. This is a limitation of CF, though, as Terraform for example has a `force_destroy` option for this purpose. Not that I'm biased :)

Reply:
😎, thank you! 

For `update-stack`
I grabbed the template using `get-template`, so that I can update the template body but with my changes, it kept giving me an error saying the template-body wasn't valid.